.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
+npx lint-staged
```

Comment:
**question:** Why switch from `yarn` to `npx`?

Reply:
I made a request to support team to have node v18.16.1 in the self service, to allow usage of 'yarn' and not 'npx' in the command husky